packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -8099,63 +8099,765 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eservices/{eserviceId}/descriptors/{descriptorId}/certifiedAttributes/groups:
+    parameters:
+      - name: eserviceId
+        in: path
+        description: The E-Service ID of the Descriptor
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: descriptorId
+        in: path
+        description: The ID of the Descriptor
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+        - eservices
+      summary: Create Certified Attributes Group for E-Service Descriptor
+      description: Create a new group of Certified Attributes for an E-Service Descriptor in Draft state
+      operationId: createEServiceDescriptorCertifiedAttributesGroup
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: string
+                format: uuid
```

Comment:
Let's extract this into a schema, usually we avoid declaring request bodies inline, something like `EServiceDescriptorAttributesGroupSeed` - same for all endpoints